larq/optimizers_test.py

```diff
@@ -82,7 +82,40 @@ def test_overlapping_predicates(self):
     def test_missing_default(self):
         with pytest.warns(Warning):
```

Comment:
Shouldn't this be?
```suggestion
        with pytest.raises(ValueError):
```

Reply:
Nope, we [warn about this](https://github.com/larq/larq/blob/master/larq/optimizers.py#L209) instead of throwing.

Reply:
I think would be for scenario for example where you "freeze" FP weights by only passing an optimizer for binary weights.

Reply:
:+1: I think I misread the comment [here](https://github.com/larq/larq/pull/446/files#diff-4930c450271ff44e3368d2a51994982eR110)

Reply:
Sorry the git diff just was weird to read.